docs/data-sources/catalog_type.md

```diff
@@ -0,0 +1,29 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "incident_catalog_type Data Source - terraform-provider-incident"
+subcategory: ""
+description: |-
+  This data source provides information about a catalog type.
+---
+
+# incident_catalog_type (Data Source)
+
+This data source provides information about a catalog type.
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Optional
+
+- `name` (String) Name is the human readable name of this type
+- `type_name` (String) The type name of this catalog type, to be used when defining attributes. This is immutable once a CatalogType has been created. For non-externally sync types, it must follow the pattern Custom["SomeName "]
```

Comment:
This should be `Custom["SomeName"]` without the trailing space

Reply:
Ohh, this is actually from our API docs 🤦 can't change here.